src/util/replaygain.h

```diff
@@ -12,9 +12,16 @@ class ReplayGain {
     static const double kRatioMin; // lower bound (exclusive)
     static const double kRatio0dB;
 
+    static const CSAMPLE kPeakUndefined;
+    static const CSAMPLE kPeakMin; // lower bound (inclusive)
+    static const CSAMPLE kPeakClip; // upper bound (inclusive) without clipping
+
     ReplayGain()
```

Comment:
We have two times ReplayGain in our source. This is a bit confusing when reading such a PR as diff. 
Can we find a better Name for this? ReplayGainData or something?